.env

```diff
@@ -0,0 +1,8 @@
+  API_TOKEN=""
+  IDS_CLIENT_SECRET=""
+  IDS_CLIENT_ID=""
+  IDS_SCOPE=""
+  API_ADDRESS="http://qa-uber.edgecast.com"
```

Comment:
Just wanted to confirm that these QA URLs are intended to be left in the environment configuration.

Reply:
These have been removed as an example and included in the README.md